ksqldb-functional-tests/src/test/resources/query-validation-tests/joins.json

```diff
@@ -921,9 +919,7 @@
         {"topic": "left_topic", "key": 3, "value": {"L1": "D"}, "timestamp": 60000}
       ],
       "outputs": [
-        {"topic": "OUTER_JOIN", "key": 0, "value": {"T_ID": 0, "TT_ID": null, "L1": "A", "L2": null}, "timestamp": 0},
         {"topic": "OUTER_JOIN", "key": 0, "value": {"T_ID": 0, "TT_ID": 0, "L1": "A", "L2": "a"}, "timestamp": 60000},
-        {"topic": "OUTER_JOIN", "key": 1, "value": {"T_ID": null, "TT_ID": 1, "L1": null, "L2": "b"}, "timestamp": 330000},
```

Comment:
As above. Also, can we add a "right outer" join result, too?

Reply:
Added